File: Assets/AltUnityTester/AltUnityDriver/Commands/OldFindObjects/FindElementsByComponent.cs
public class FindElementsByComponentDriver : AltBaseCommand
{
    string componentName;
    string assemblyName;
    string cameraName;
    bool enabled;
    public FindElementsByComponentDriver(SocketSettings socketSettings, string componentName, string assemblyName, string cameraName, bool enabled) : base(socketSettings)
    {
        this.componentName = componentName;
        this.assemblyName = assemblyName;
        this.cameraName = cameraName;
        this.enabled = enabled;
    }
    public System.Collections.Generic.List<AltUnityObject> Execute()
    {
        Socket.Client.Send(toBytes(CreateCommand("findObjectsByComponent", assemblyName, componentName, cameraName, enabled.ToString())));
        string data = Recvall();
        if (!data.Contains("error:")) return Newtonsoft.Json.JsonConvert.DeserializeObject<System.Collections.Generic.List<AltUnityObject>>(data);
        HandleErrors(data);
        return null;
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/OldFindObjects/find_elements_by_component.py
from altunityrunner.commands.command_returning_alt_elements import CommandReturningAltElements
class FindElementsByComponent(CommandReturningAltElements):
    def __init__(self, socket,request_separator,request_end,appium_driver,component_name,assembly_name,camera_name,enabled):
        super().__init__(socket,request_separator,request_end,appium_driver)
        self.component_name=component_name
        self.assembly_name=assembly_name
        self.camera_name=camera_name
        self.enabled=enabled
    
    def execute(self):
        if self.enabled==True:
            data = self.send_data(self.create_command('findObjectsByComponent',self.assembly_name, self.component_name , self.camera_name ,'true'))
        else:
            data = self.send_data(self.create_command('findObjectsByComponent',self.assembly_name, self.component_name , self.camera_name ,'false'))
        return self.get_alt_elements(data)